components/Dialogs/src/MDCAlertController.h

```diff
@@ -119,6 +119,11 @@
  */
 @property(nonatomic, nullable, copy) NSString *title;
 
+/**
+ A custom accessibility label for the title.
```

Comment:
Please describe the behavior when this property is nil. E.g.

```
When nil, the title's accessibilityLabel will be set to the value of @c title.
```

Reply:
Done